scripts/run-all-tests.sh

```diff
@@ -1,7 +1,8 @@
-#!/bin/bash
+#!/bin/sh
 
-ACCESS_TOKEN=$(PASSWORD_STORE_DIR=$HOME/.paas-pass pass compose/dev/access_token)
-
-export ACCESS_TOKEN
+if [ -z $ACCESS_TOKEN ]; then 
```

Comment:
I wonder if we should call this `COMPOSE_ACCESS_TOKEN` if we're expecting it to be passed in externally, to make it clear what it belongs to and not conflict with access tokens from any other provider?